examples/site.yml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: all
+
+  pre_tasks:
+    - name: Update APT cache
+      apt: update_cache=yes
+
+  roles:
+    - { role: "azavea.gunicorn" }
+    - { role: "gunicorn.example" }
```

Comment:
I'd probably wire up `azavea.gunicorn` as a dependency of `gunicorn.example`. Mainly because that's probably how we would wire things up when using this role.